libs/sam/src/build/get-entries-from-cloudformation.ts

```diff
@@ -52,11 +55,14 @@ function getEntry(
 function getEntryForFunction(
     properties: { [key: string]: any },
     options: ExtendedBuildBuilderOptions,
-    srcMapInstall: string
+    srcMapInstall: string,
+    globalProperties?: { [key: string]: any },
```

Comment:
I was about to suggest not using `any` here, but realised that I'd done the same. As you were.

Reply:
yeah it's all or nothing isn't.  i'd prefer types but it's fairly big to cover all potentials